noir-projects/noir-contracts/contracts/amm_contract/src/config.nr

```diff
@@ -27,3 +30,17 @@ impl Deserialize<CONFIG_LENGTH> for Config {
         }
     }
 }
+
+impl Packable<CONFIG_LENGTH> for Config {
+    fn pack(self: Self) -> [Field; CONFIG_LENGTH] {
+        [self.token0.to_field(), self.token1.to_field(), self.liquidity_token.to_field()]
+    }
+
+    fn unpack(fields: [Field; CONFIG_LENGTH]) -> Self {
+        Self {
+            token0: AztecAddress::from_field(fields[0]),
+            token1: AztecAddress::from_field(fields[1]),
+            liquidity_token: AztecAddress::from_field(fields[2]),
+        }
+    }
```

Comment:
Just use serialize

Reply:
Same as above. Packable required for public storage